components/engine_rocks/src/cf_options.rs

```diff
@@ -27,28 +29,40 @@ impl CFOptionsExt for RocksEngine {
     }
 }
 
-#[derive(Clone)]
-pub struct RocksColumnFamilyOptions(RawCFOptions);
+#[derive(Default, Clone)]
+pub struct RocksCfOptions(RawCfOptions);
```

Comment:
Bit weird to have both `RocksCfOptions` and `CFOptionsExt`/`RocksDBOptions`.

Reply:
I will rename all those struct in a separate PR.